DatadogSessionReplay/Sources/Recorder/Recorder.swift

```diff
@@ -129,8 +139,15 @@ public class Recorder: Recording {
                 context: .init(recorderContext.applicationID)
             )
         } catch let error {
+            isSuccesful = false
             telemetry.error("[SR] Failed to take snapshot", error: DDError(error: error))
         }
+        telemetry.stopMethodCalled(methodCalledTrace, isSuccessful: isSuccesful)
+    }
+
+    private enum MethodCallConstants {
+        static let captureRecordOperationName = "Capture Record"
+        static let className = "Recorder"
```

Comment:
suggestion/ There is no "SR" context anywhere. WDYT of following?
```suggestion
        static let captureRecordOperationName = "Capture Record"
        static let className = "SR.Recorder"
```

Reply:
Yeah - probably worth adding package information. I'll try to find a common way to the class signature with the package name.

Since it's slow I'll use lazy static var within to reference.